src/rgw/rgw_user.cc

```diff
@@ -2628,9 +2628,9 @@ int RGWUserMetadataHandler::do_put(RGWSI_MetaBackend_Handler::Op *op, string& en
                                    RGWMetadataObject *obj,
                                    RGWObjVersionTracker& objv_tracker,
                                    optional_yield y,
-                                   RGWMDLogSyncType type)
+                                   RGWMDLogSyncType type, bool from_remote_zone)
 {
-  RGWMetadataHandlerPut_User put_op(this, op, entry, obj, objv_tracker, y, type);
+  RGWMetadataHandlerPut_User put_op(this, op, entry, obj, objv_tracker, y, type, false /* from_remote_zone */);
```

Comment:
why doesn't this pass on the `bool from_remote_zone` parameter?

Reply:
@cbodley I set it to `false` to make what we are passing, obvious. But if that is bad practice I can change it.

Reply:
sorry, github didn't quite show enough context here:
```c++
int RGWUserMetadataHandler::do_put(RGWSI_MetaBackend_Handler::Op *op, string& entry,
                                   RGWMetadataObject *obj,
                                   RGWObjVersionTracker& objv_tracker,
                                   optional_yield y,
                                   RGWMDLogSyncType type)
                                   RGWMDLogSyncType type, bool from_remote_zone)
{
  RGWMetadataHandlerPut_User put_op(this, op, entry, obj, objv_tracker, y, type);
  RGWMetadataHandlerPut_User put_op(this, op, entry, obj, objv_tracker, y, type, false /* from_remote_zone */);
```
this is throwing away the value passed into `do_put()`

Reply:
@cbodley ack

Reply:
fixed